recipes/smallgenomeutilities/0.3.3/meta.yaml

```diff
@@ -46,6 +48,7 @@ test:
     - extract_coverage_intervals -h
     - extract_sam -h
     - extract_seq -h
+    - frameshift_deletions_checks -h
```

Comment:
[`command not found`](https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/49261/workflows/fa860f70-0584-4427-a3e6-8a7c87b407d4/jobs/160754?invite=true#step-106-633)